### PR TITLE
Feat/design

### PR DESCRIPTION
**Pull request by manish591**
### Added UI part of video library 

Features/Pages Included: 
* Video Listing page
* Filter By Category
* Playlist Management
* Watch Later Page
* Liked Video Page
* History page

ScreenShot: 

Desktop Design:
image

Tablet Design: 
mobile (1)

Mobile Design: 
mobile

### CODE REVIEW

File: src/components/card/PlaylistCard.jsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import "./PlaylistCard.css";
+
+const PlaylistCard = () => {
+  return (
+    <div className="playlist-card">
+      <section className="playlist-card__image-container">
+        <img
+          src="https://images.unsplash.com/photo-1534080564583-6be75777b70a?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NzN8fGZvb2R8ZW58MHwwfDB8fA%3D%3D&auto=format&fit=crop&w=500&q=60"
+          alt=""
```
Comment (mansipadgelwar) [inline src/components/card/PlaylistCard.jsx:10]: alt tag is empty

**Review by mansipadgelwar — APPROVED** [review]
UI looks clean.

**Review by shrey27 — APPROVED** [review]
UI looks great in both desktop and mobile view
PR is ready to merge

File: src/components/card/PlaylistCard.jsx (hunk 1 above)
Comment (manish591) [reply src/components/card/PlaylistCard.jsx:10]: I will add it

**Pull request merged**